Modules/HR/Http/Controllers/TagsController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Modules\HR\Http\Controllers;
+
+use Illuminate\Routing\Controller;
+use Illuminate\Http\Request;
+use App\Models\Tag;
+
+class TagsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $attr['tags'] = Tag::orderBy('tag_name')->get();
+
+        return view('hr::tags.index')->with($attr);
+    }
+
+    public function store(Request $request)
```

Comment:
Add backend validation as well. For this you need to create a file with name `TagRequest` inside Request folder. You can take reference form other Request files. Then you can use that validation file here like. Also use TagRequest while updating. So try to make that file generic, so that it can be used in both the operation.
```suggestion
    public function store(TagRequest $request)
```

Reply:
Ok, I'll make the requested changes.